guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -66,13 +73,20 @@ To update the certificates on a currently running Katello installation, run:
     --certs-update-server --certs-update-server-ca
 ----
 endif::[]
-
++
+Note that you must not access or modify `/root/ssl-build`.
+ifdef::satellite[]
+Contact Red Hat Support for further assistance.
+endif::[]
 . From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that installs a new {Project} with custom SSL certificates or updates certificates on a currently running {Project}.
 +
 If you are unsure which command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists.
 If the file exists, run the second `{foreman-installer}` command that updates certificates.
 +
-IMPORTANT: Do not delete the certificate archive file after you deploy the certificate.
+[IMPORTANT]
+====
+Do not modify or delete the certificate archive file after you deploy the certificate.
 It is required, for example, when upgrading {ProjectServer}.
```

Comment:
Maybe also reword to "foreman-installer needs it" as above?

Reply:
Reworded :+1: